rasa_core/server.py

```diff
@@ -448,6 +453,27 @@ def load_model():
         logger.debug("Finished loading new agent.")
         return '', 204
 
+    @app.route("/evaluate",
+               methods=['POST', 'OPTIONS'])
+    @requires_auth(app, auth_token)
+    @cross_origin(origins=cors_origins)
+    def evaluate_stories():
+        """Evaluate stories against the currently loaded model."""
+
+        data = convert_bytes_to_string(request.get_data())
+        tmp = tempfile.NamedTemporaryFile(delete=False)
```

Comment:
I think we should use 
```
import rasa_nlu

rasa_nlu.utils.create_temporary_file(data)
```
in this case we also won't need the `convert_bytes_to_string`